animation-frame-rate/README.md

```diff
@@ -0,0 +1,130 @@
+
+## Authors:
+
+- [Antoine Quint](https://github.com/graouts)
+- [Simon Fraser](https://github.com/smfr)
+
+## Participate
+- https://github.com/WebKit/explainers
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [tl;dr](#tldr)
+- [Introduction](#introduction)
+- [Variable frame rate](#variable-frame-rate)
+- [Use cases](#use-cases)
+- [General proposal](#general-proposal)
+- [Proposed changes to existing technologies](#proposed-changes-to-existing-technologies)
+  - [Web Animations](#web-animations)
+  - [CSS Animations and CSS Transitions](#css-animations-and-css-transitions)
+  - [JavaScript Animations](#javascript-animations)
+- [Privacy considerations](#privacy-considerations)
+- [Considered alternatives](#considered-alternatives)
+- [Acknowledgements](#acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## tl;dr
+
+We propose adding a `frameRate` property on the `Animation` interface to allow authors to run animations at a higher or lower frame rate than the default frame rate the browser uses to [update the page rendering](https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering).
+
+## Introduction
+
+Plenty of devices sport displays with refresh rates that are greater than 60Hz; mobile devices exist with 90Hz and 120Hz refresh rates, and desktop screens exist with 144Hz and 240Hz refresh rates.
+
+Starting with the iPad Pro in 2018 and now in 2021 with the iPhone 13 Pro and the 14” and 16” 2021 MacBooks Pro, Apple devices have supported multiple refresh rates under the moniker [ProMotion](https://developer.apple.com/documentation/quartzcore/optimizing_promotion_refresh_rates_for_iphone_13_pro_and_ipad_pro). Using this technology, displays may refresh their display at up to 120Hz, but can also adjust to various refresh rates along the way, instead of using the constant refresh rate of 60Hz.
+
+On Apple's 120Hz devices, accelerated already run at 120Hz thanks to Core Animation's built-in support for the ProMotion technology, whereas the rest of the Web page only updates at 60Hz. This "rest of the Web page" includes script-driven animations using `requestAnimationFrame()` as well as all non-accelerated Web Animations (including CSS Transitions and CSS Animations).
+
+WebKit chose to restrict web content updates to 60Hz for two reasons: first, we measured a significant increase in power usage, and second, we found several examples of web pages that had incorrect behavior when `requestAnimationFrame()` callbacks were fired at a non-60Hz frequency.
+
+However, now that there are more devices with high refresh rate displays, we believe it's judicious to propose advancing the Web platform to take advantage of higher refresh rates and allow authors to choose a suitable refresh rate for different kinds of animations.
```

Comment:
We want content to be able to take advantage of lower refresh rates too, right?

Reply:
Absolutely, this is very important. The primary reason this work is happening is because of higher frame rates, but it's an opportunity to improve power usage by letting authors express the existence of lower-impact animations.